Dockerfile-7.3

```diff
@@ -49,7 +49,7 @@ RUN \
 
 # Setup fpm paths and log
 RUN \
-  mkdir /run/php/ && \
+  mkdir -p /run/php/ && \
```

Comment:
I guess the `-p` should be added in 7.2, 7.1, etc as well? The test builds seem to fail with `mkdir: cannot create directory ‘/run/php/’: File exists`

Reply:
Yup, seems like it, I'll add it

Reply:
:+1: